### DISCUSSION TITLE
io/file-path

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#135] [Announcements]
# io/file-path

Java程序员进阶之路，小白的零基础Java教程，详解 File、Path、Paths、Files，操作文件不再难

https://tobebetterjavaer.com/io/file-path.html

**Comment by sirmingwang** [comment]
5）目录的遍历
String[] list() ：返回一个 String 数组，表示该 File 目录中的所有子文件或目录。

***此处应该没有目录，只遍历文件吧

**Comment by dikk123** [comment]
请问作者一个问题，我觉得这一节的名字，是不是改成文件类更加合适呢？
因为基本上说的都是java.io.File这个类，而不是文件流哦。
我起初一度被这个标题和内容所困惑。。。。
